src/ngraph_builder.cc

```diff
@@ -1597,7 +1665,9 @@ static Status TranslateDepthwiseConv2dNativeOp(
                                           input_shape[3]};
     auto ng_sliced_input =
         make_shared<ng::op::Slice>(ng_input, lower_bound, upper_bound);
-
+    if (!is_nhwc) {
+      SaveNgOp(first_ng_op_map, op->name(), ng_input);
+    }
```

Comment:
same as above

Reply:
ng_filter and ng_input are different data flows, so this check I think is still valid